packages/forms/src/UIForm/fields/Code/Code.component.js

```diff
@@ -32,42 +45,79 @@ try {
 			this.props.onFinish(event, { schema: this.props.schema });
 		}
 
+		onKeyDown(event) {
+			if (event.keyCode === keyCode.codes.esc) {
+				const now = Date.now();
+				if (this.lastEsc && this.lastEsc - now < 1000) {
```

Comment:
why not put it in the component state ?
if not why not initializing the property on component instantiation ?

Reply:
ok I add you that with a comment above it